### ISSUE TITLE
Dark fork

### ISSUE THREAD

**Issue by BruceMustache** [MysticTempest/REFI_Textures#1]
I love REFI_Textures but the only that I don't like so much is the white interface
So I suggest a dark fork

**Comment by MysticTempest** [comment]
Thanks! Glad to hear you really enjoy this texture pack.

As for the dark theme. 
Well, there's only about 30 or so gui/inventory textures that affect the interface. So, I don't plan on creating a separate fork for such a small change.
As a bit of a compromise though; I went ahead, and made some alternate textures. 

They've been placed here:
https://github.com/MysticTempest/REFI_Textures/tree/master/experimental_dark_theme



But, so they won't affect the default light-theme interface, they've been zipped up.

Per the readme in the 'experimental_dark_theme' folder; just extract the zip file. Then merge the extracted folder with the other 'textures' folder in the pack.
Everything should overwrite correctly.

Lastly, do take note of the readme's bottom section about tweaking MineClone2's in-game text color to compliment the dark theme.

**Comment by BruceMustache** [comment]
Thanks, can I make a fork of these?

**Comment by MysticTempest** [comment]
> Thanks, can I make a fork of these?

No prob!
And, sure! The texture pack is licensed under a CC BY-SA 4.0 license; you're free to do so.

**Comment by BruceMustache** [comment]
I make that fork and push the changes extracting the zip
What I do now to put it on ContentDB?

**Comment by MysticTempest** [comment]
> I make that fork and push the changes extracting the zip https://github.com/BruceMustache/REFI_Textures_Dark
> What I do now to put it on ContentDB?

Ah, sorry; just saw your follow up comment.

I've been working on a slightly better way to support both themes via some user runnable scripts that will automate the installation of either theme. While still defaulting to a light theme.
But, feel free to keep your default dark theme fork.

But, I was going to add that I missed a couple textures the first round.
The dark textures for Xdecor's enchanting table, and the square minimap texture.

[dark_theme_additional_textures.zip](https://github.com/MysticTempest/REFI_Textures/files/4975254/dark_theme_additional_textures.zip)

As for adding your fork to ContentDB. At a minimum; you'll want to update the "description.txt" and "texture_pack.conf" files. 
As it might cause name conflict issues.
Other than that. You should be okay to upload your fork.
But, I'd recommend checking in the official Minetest forums if you have further issues or concerns with ContentDB.

**Comment by BruceMustache** [comment]
Thanks, I added that dark textures in my dark fork (https://github.com/BruceMustache/REFI_Textures_Dark/commit/95a07e6e8736dc4dadd35cb493e67f931a54a372)
And change description.txt as well as texture_pack.conf (https://github.com/BruceMustache/REFI_Textures_Dark/commit/a900beda6db06850668db6c39a8a7f9f0d8ed49a)
I'm watching this repository to update the fork whenever you update

**Comment by MysticTempest** [comment]
I just pushed a slight rework of my experimental dark_theme support.

For an alternative fix to name collisions; I renamed the dark theme, and light theme files. Then wrote scripts that can be executed to install the desired theme.
It also updates the in-game screenshot to reflect the theme.

Re-closing now, as this is a better way for me to support it than the zip file.
Committed here:
https://github.com/MysticTempest/REFI_Textures/commit/55a581601057a497f91bf293f4c70f2dc15621a0
